packages/cw-umee-types/src/msg.rs

```diff
@@ -24,128 +23,118 @@ pub enum UmeeMsg {
 #[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
 #[serde(rename_all = "snake_case")]
 pub struct StructUmeeMsg {
-  assigned_msg: u16,
+  assigned_msg: MsgTypes,
   supply: Option<SupplyParams>,
   withdraw: Option<WithdrawParams>,
+  max_withdraw: Option<MsgMaxWithDrawParams>,
   collateralize: Option<CollateralizeParams>,
   decollateralize: Option<DecollateralizeParams>,
   borrow: Option<BorrowParams>,
+  max_borrow: Option<MsgMaxBorrowParams>,
   repay: Option<RepayParams>,
   liquidate: Option<LiquidateParams>,
+  supply_collateral: Option<SupplyCollateralParams>,
 }
 
+fn default_struct_umee_msg(m: MsgTypes) -> StructUmeeMsg {
+  StructUmeeMsg {
+    assigned_msg: m,
+    supply: None,
+    withdraw: None,
+    collateralize: None,
+    decollateralize: None,
+    borrow: None,
+    max_borrow: None,
+    repay: None,
+    liquidate: None,
+    max_withdraw: None,
+    supply_collateral: None,
```

Comment:
tbh, I don't like this placeholder style struct. I don't know if there is better way though. Last year, I remember we were supposed to check it.